projects/openvswitch/fuzzer.options

```diff
@@ -0,0 +1,3 @@
+[libfuzzer]
+close_fd_mask = 3
+dict = ovs.dict
```

Comment:
Isn't one of these targets just a JSON parser? Is ovs.dict meaningful for it?
Is some of the RPC stuff in the dictionary?

Reply:
Ack. Will add a OVS specific json dictionary, although I am sure afl's json dict might be useful as well.